tests/candev/Makefile

```diff
@@ -4,13 +4,19 @@ USEMODULE += shell
 USEMODULE += can
 USEMODULE += isrpipe
 
-FEATURES_OPTIONAL += periph_can
 
 # define the CAN driver you want to use here
-CAN_DRIVER ?= CAN_EXT
+CAN_DRIVER ?= NATIVE
 
-ifeq ($(CAN_DRIVER), CAN_EXT)
-# external CAN driver modules/variables go here
+ifeq ($(CAN_DRIVER), NATIVE)
+
+	FEATURES_OPTIONAL += periph_can
+
+else ifeq ($(CAN_DRIVER), MCP2515)
+	USEMODULE += mcp2515
+
+else ifeq ($(CAN_DRIVER), CAN_ALT)
+	# other can modules can be defined here
 
 endif
```

Comment:
```suggestion
ifeq ($(CAN_DRIVER), PERIPH_CAN)
  FEATURES_REQUIRED += periph_can
else ifeq ($(CAN_DRIVER), MCP2515)
  USEMODULE += mcp2515
else ifeq ($(CAN_DRIVER), CAN_ALT)
  # other can modules can be defined here
endif
```

Reply:
I think this would match better the current status.

Reply:
I applied your suggestion, with the addition of an additional check on the used board. If the board is native, it will always default to periph_can, and not allow a different driver. 
Why? Well... because before it was pretty clear that you had to use can_native on the native board. With the recent changes in the can structure it has (imo) become a little bit confusing that you have to use can_periph now for native. 
So for users who are unfamiliar with can in RIOT, I think forcing this option on them will save them a lot of headaches I think :)